conformance/Makefile

```diff
@@ -1,7 +1,8 @@
 NKG_TAG = edge
 NKG_PREFIX = nginx-kubernetes-gateway
 GATEWAY_CLASS = nginx
-SUPPORTED_FEATURES = Gateway,HTTPRoute
+SUPPORTED_FEATURES = HTTPRoute,HTTPRouteQueryParamMatching,HTTPRouteMethodMatching,HTTPRoutePortRedirect,HTTPRouteSchemeRedirect
```

Comment:
Is this going to be updated constantly for all new features added? i.e. will we have to include additional A/C for stories to update this?

Reply:
Until we support all features, yes. But that's probably a long way out because that would include Mesh features.

Reply:
Where is the list of features in the gateway repo? For reference.